### DISCUSSION TITLE
设置logo不生效

### DISCUSSION THREAD

**Discussion by Naturalvolume** [umijs/dumi#2157] [Q&A]
![image](https://github.com/umijs/dumi/assets/50625993/2a9ef5fe-8fec-49da-8ee1-8740f7f9fa1d)
![image](https://github.com/umijs/dumi/assets/50625993/333dceb5-a004-4e3b-96b2-932c7b409336)
把logo放到public文件夹中，在配置中引入logo，但logo设置并没有生效。

**Comment by PeachScript** [comment]
请仔细阅读文档：https://d.umijs.org/config#logo
